src/Controller/CreateProjectController.php

```diff
@@ -135,19 +133,37 @@ public function createProject(Request $request): Response
      */
     public function submitted(Request $request): Response
     {
+        $formData = $_SESSION['form_data'];
+
+        $jiraUrl = $this->getParameter('app.project_tracker_url');
+        $jiraUrl = is_string($jiraUrl) ? $jiraUrl : '';
+        $tempoTeamId = $formData['selectedTeamConfig']['tempo_team_id'] ?? '';
```

Comment:
Is it handled if `jiraUrl` (also not fan of hardcoding relation to jira here, we are already looking at other trackers) if it's an empty string (same for tempoTeamId, may not be named tempo in 6-8 months).

Reply:
So maybe these setting should come from the `ApiServiceInterface->getEndpoints()`

Reply:
Will add getEndpoints() to in